src/aleph/http/server.clj

```diff
@@ -634,7 +709,12 @@
          :exception-caught
          ([_ ctx ex]
           (when-not (instance? IOException ex)
-            (log/warn ex "error in websocket handler"))
+            (if (some? error-logger)
+              (try
+                (error-logger ex)
+                (catch Throwable e
+                  (log/warn "exception in error logger" e)))
```

Comment:
Do you ignore the stacktrace for this exception on purpose? What I mean is that `(log/warn e "error in error logger")` would print both the exception message and its trace, but your code will only print the message itself.